### PR TITLE
Update Docs

### LINKED ISSUE #64: Update Docs
Create a docs folder and move all .md files inside it

and update the files as follows 
**PR_template.md**
rename to pull_request_template.md
add a comment in Fixes issue, "Use one of fixes or addresses and remove the other one"

**README.md**
Remove How to start contributing section

Inside Important Notice add this line and add relative links to both files 
Before contributing to this project do check CONTRIBUTING.md file and CODE_OF_CONDUCT.md file.

### **UPDATE ANY RELATIVE LINK(image,file) THAT GETS AFFECTED BY MOVING FILES TO NEW DIRECTORY**

In case of any doubt,ask!

### PR DESCRIPTION
**Pull request by Adyasha8105**
This PR closes #64.

### CODE REVIEW

File: Docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,32 +1,36 @@
 ## Fixes issue
+
 Closes: #issue_number
 Addresses: #issue_number (if multiple persons are assigned, the issue)
+Use one of fixes or addresses and remove the other one.
```
Comment (rajats98) [inline Docs/PULL_REQUEST_TEMPLATE.md]: Add this as a comment, like this
 \

File: Docs/README.md [hunk 2]
```diff
@@ -39,61 +40,17 @@ This application takes an image of the leaf or crop as input and shows the resul
...
-
 ## Important Notice
 
 **Crop Ai** has been selected at [GirlScript Summer of Code 2020](https://www.gssoc.tech/)!!!
 
 >Only GSSoC participants are allowed to contribute to this repository.
 
 <p align="center">
-  <img width="300" height="300" src="./designs/logo-designs/gssoc-logo.png">
+  <img width="300" height="300" src="../designs/logo-designs/gssoc-logo.png">
```
Comment (rajats98) [inline Docs/README.md]: Resize this to 200 width and 200 height

File: Docs/README.md [hunk 3]
```diff
@@ -39,61 +40,17 @@ This application takes an image of the leaf or crop as input and shows the resul
...
-```terminal
-git push -u origin <your_branch_name>
-```
-
-**8.** Create a Pull Request!
-
-**Congratulations!**  :boom: You have made your first contribution
- 
-
+Before contributing to this project do check CONTRIBUTING.md file and CODE_OF_CONDUCT.md file.
```
Comment (rajats98) [inline Docs/README.md]: Add links to CONTRIBUTING.md and CODE_OF_CONDUCT.md , on clicking file name the respective file should open.

**Review by rajats98 — CHANGES_REQUESTED** [review]
Rename Docs to docs
rename PULL_REQUEST_TEMPLATE.md to pull_request_template.md

**Comment by Adyasha8105** [comment]
@rajats98 Can you check my PR?

**Comment by rajats98** [comment]
also do this, it was also there in last review, maybe you missed it 

Rename Docs to docs
rename PULL_REQUEST_TEMPLATE.md to pull_request_template.md

**Comment by Adyasha8105** [comment]
@rajats98 Now you can check my PR?

**Pull request merged**